doc/source/cli_topo.md

```diff
@@ -0,0 +1,265 @@
+
+      - root
+        ├─ tables
+        │   ├─ elev_stats.csv
+        │   └─ elev_with_mask_stats.csv
+        ├─ plots
+        │   ├─ elev_map.png
+        │   └─ terrain_attributes_map.png
+        ├─ rasters
+        │   ├─ aspect.tif
+        │   ├─ curvature.tif
+        │   ├─ hillshade.tif
+        │   ├─ rugosity.tif
+        │   ├─ slope.tif
+        │   ├─ terrain_ruggedness_index.tif
+        │   └─ ...
+        ├─ report.html
+        ├─ report.pdf
+        └─ used_config.yaml
+```
+
+## Report example
+
+```{eval-rst}
+    .. raw:: html
+
+        <meta charset='UTF-8'><title>Topographic summary results</title></head>
+
+        <h2>Elevation Model</h2>
+        <img src='_static/elevation (m).png' alt='Image PNG' style='max-width: 100%; height: auto;'>
+        <h2>Masked elevation Model</h2>
+        <img src='_static/masked_elevation.png' alt='Image PNG' style='max-width: 100%; height: auto;'>
+        <div style='clear: both; margin-bottom: 30px;'>
+        <h2>Information about inputs</h2>
+        <table border='1' cellspacing='0' cellpadding='5'>
+        <tr><th>Information</th><th>Value</th></tr>
+        <tr><td>reference_elev</td><td>{'path_to_elev': '/xdem/examples/data/Longyearbyen/data/DEM_1990.tif', 'path_to_mask': '/xdem/examples/data/Longyearbyen/data/glacier_mask/CryoClim_GAO_SJ_1990.shp', 'force_source_nodata': -9999, 'from_vcrs': 'EGM96', 'to_vcrs': 'EGM96'}</td></tr>
+        </table>
+        </div>
+        <div style='clear: both; margin-bottom: 30px;'>
+        <h2>elevation information</h2>
+        <table border='1' cellspacing='0' cellpadding='5'>
+        <tr><th>Information</th><th>Value</th></tr>
+        <tr><td>Driver</td><td>GTiff</td></tr>
+        <tr><td>Filename</td><td>/xdem/examples/data/Longyearbyen/data/DEM_1990.tif</td></tr>
+        <tr><td>Grid size</td><td>us_nga_egm96_15.tif</td></tr>
+        <tr><td>Number of band</td><td>(1,)</td></tr>
+        <tr><td>Data types</td><td>float32</td></tr>
+        <tr><td>Nodata Value</td><td>-9999</td></tr>
+        <tr><td>Pixel interpretation</td><td>Area</td></tr>
+        <tr><td>Pixel size</td><td>(20.0, 20.0)</td></tr>
+        <tr><td>Width</td><td>1332</td></tr>
+        <tr><td>Height</td><td>985</td></tr>
+        <tr><td>Transform</td><td>| 20.00, 0.00, 502810.00|
+        | 0.00,-20.00, 8674030.00|
+        | 0.00, 0.00, 1.00|</td></tr>
+        </table>
+        </div>
+        <div style='clear: both; margin-bottom: 30px;'>
+        <h2>Global statistics</h2>
+        <table border='1' cellspacing='0' cellpadding='5'>
+        <tr><th>Information</th><th>Value</th></tr>
+        <tr><td>mean</td><td>381.32</td></tr>
+        <tr><td>median</td><td>365.23</td></tr>
+        <tr><td>max</td><td>1022.29</td></tr>
+        <tr><td>min</td><td>8.38</td></tr>
+        <tr><td>sum</td><td>500301600.0</td></tr>
+        <tr><td>sumofsquares</td><td>268858540032.0</td></tr>
+        <tr><td>90thpercentile</td><td>727.55</td></tr>
```

Comment:
The stats name could be written in plain text (e.g. "90th percentile", or "LE90") rather than using the keys, but that's not super important if too annoying.